### PR TITLE
Showing definition with changes on ContributePrompt

### LINKED ISSUE #617: Workflow: Multiselect - In the Contribution dialog show the list of components with changes/being contributed
From the [comment](https://github.com/clearlydefined/website/pull/508#issuecomment-474026332) in #508

### PR DESCRIPTION
**Pull request by storrisi**
This is how the ContributePrompt would look:

Schermata 2019-05-16 alle 17 37 16

Fixes #617

### CODE REVIEW

File: src/components/ContributePrompt.js [hunk 1]
```diff
@@ -82,6 +82,24 @@ export default class ContributePrompt extends Component {
...
+  renderEditedDefinitions = () => {
+    const { definitions } = this.props
+    return (
+      <FormGroup>
+        <ControlLabel>Involved Components</ControlLabel>
+        <ul className="definitions-list">
+          {definitions &&
+            definitions.map(definition => (
+              <li>
+                <span className="definition-name">{definition.name}</span>
```
Comment (geneh) [inline src/components/ContributePrompt.js]: Can we add the namespace and the type logo too?

Comment (storrisi) [reply src/components/ContributePrompt.js]: I think this is a great suggestion. Will do.

**Comment by storrisi** [comment]
I've added logo and namespace has suggested

File: src/components/ContributePrompt.js [hunk 2]
```diff
@@ -84,17 +85,26 @@ export default class ContributePrompt extends Component {
 
   renderEditedDefinitions = () => {
     const { definitions } = this.props
+    console.log(definitions)
```
Comment (geneh) [inline src/components/ContributePrompt.js]: Should console.log be removed here and down below?

**Comment by geneh** [comment]
👍

**Pull request merged**